Common/Cloud-Account/aws-tf-connectors-automation/connectors.tfvars

```diff
@@ -0,0 +1,7 @@
+region = "us-east-1"
```

Comment:
I would explain that this file needs to be updated with the unique values for deployment to work.